### DISCUSSION TITLE
Added a "Install on Tolino Devices" section in the wiki

### DISCUSSION THREAD

**Discussion by hasezoey** [koreader/koreader#9521] [General]
The Page can be found at https://github.com/koreader/koreader/wiki/Installation-on-Tolino-devices (also in the sidebar)

in "extra notes" i had written that a HomeScreen APK and a Button APK should also be installed, in my case i use the following:
- SmartLauncher for HomeScreen
- ButtonSavior (non root) for Buttons

i did not include them in the wiki itself, because i didnt know if listing other (optional and non-FOSS) applications would be OK

i also would like some feedback from the maintainers if the wiki page is correctly set-up or if anything needs to be added
cc @pazos @Frenzie @NiLuJe 

Edit:
The Wiki Entry will be removed by choice of maintainers, but i have "outsourced" it into a gist which can be read at https://gist.github.com/hasezoey/d16ba0f980f00cd2193132afe4714c7c for anyone searching

**Comment by pazos** [comment]
I have a very strong opinion on this topic.

After all tolinos **are** android. And KOReader is **yet another android app** under them.
I don't think it is our duty to cover specific issues with **any** android device. Android instructions are ok as is.

We've had a few android users using Tolinos wihout problems. I don't know how they managed to do the job and I couldn`t care less. The assumption is: you need to be able to install APKs to install KOReader on android. It doesn't matter if it is some closed Sony/Tolino/Nook/...

There're external resources that are useful and people that can help elsewhere. We just provide an APK and handle issues within it.

**Reply by hasezoey** [reply]
i thought a guide would be good because tolino requires special setup to be able to be used as a android device, and help really is only available in the form of german (at least forums) and information is spread about and hard to access

also in my opinion KOReader is way better than the standard tolino reader app (especially the performance of page turning and image processing and overall responsiveness)

**Reply by pazos** [reply]
I understand the intent. But IMO this is not the best place. We also don't cover prerequisites on other platforms extensively. Some examples: jailbreak on kindles, developers firmware on cervantes, rooting on Sony-DPT_R1, rooting on Nooks...

The prerequisites covered on android instructions are generic and apply to all android devices: enable installation from apps outside the Play Store. The different install methods are also generic.

Specific instructions that are only required for a very limited subset of devices are better elsewhere. For instance see https://github.com/HappyZ/dpt-tools.

Also not having **any** documentation at all for specific quirks of specific devices under the android platform is better than have different levels of documentation for different devices that may be out of date, updated but can't be applied to older devices, etc. These are a PITA to maintain.

And we're just an android app. No more, no less.

**Reply by hasezoey** [reply]
> And we're just an android app. No more, no less.

said about the app that tries to control the system properties through workarounds /s

<sub>if this is not clear, this is meant to be a joke</sub>

---

i definitely understand what you are trying to say, and i think it was at least work a try, you can remove it again if you want to and i will put it up on a personal (public) gist, in the hopes it helps people

**Comment by pazos** [comment]
> i definitely understand what you are trying to say, and i think it was at least work a try, you can remove it again if you want to and i will put it up on a personal (public) gist, in the hopes it helps people

Please do that. I'll remove it from the wiki.

I appreaciate the work/effort you're putting on your specific device/brand. Other users will definitly benefit from it.

Please don't be discouraged to contribute because some random maintainer (me) won't agree with you in some aspects. There're plenty where we can reach an agreeement ;)

**Reply by hasezoey** [reply]
> Please don't be discouraged to contribute because some random maintainer (me) won't agree with you in some aspects. There're plenty where we can reach an agreeement ;)

i mean it is open-source and as long as the developers are not problematic, it is always worth a try to create a PR / issue / "official documentation"

**Reply by hasezoey** [reply]
as a note, i have updated the description of the discussion to have a link to the gist noting that the wiki entry will be removed
TL;DR: the gist guide is at https://gist.github.com/hasezoey/d16ba0f980f00cd2193132afe4714c7c

**Reply by Frenzie** [reply]
I've deleted the page.

**Reply by hasezoey** [reply]
as a note, because the page is deleted, i have removed the entry from the sidebar
